### PR TITLE
#3 feature: build signup endpoint

### LINKED ISSUE #13: Eslint not working

### PR DESCRIPTION
**Pull request by bryan936M**
#### What does this PR do?
- make a buyer signup functionality
#### Description of Task to be completed?
- create a signup route
- create a signup controller
- adjust user database table to suit requirements
#### How should this be manually tested?
- clone this repo
- after opening the repo locally, install dependencies
- run the following command in a terminal: npm run sev
- open postman and go to: 'http://localhost:3000/auth/buyer-signup'
- input a user's credentials. Ex:
- {"name": "sample_user ","email":"sample_user@email.com", "password": "sample_userPassword"}
#### What are the relevant pivotal tracker/Trello stories?
- visit my trello card: 'https://trello.com/c/xpXq1R1r'

### CODE REVIEW

**Review by faid-terence — APPROVED** [review]
Great Work @bryan936M

**Review by vanessa-abewe — APPROVED** [review]

**Review by vanessa-abewe — APPROVED** [review]

File: src/controller/authController.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const hashedPassword = await hasher(password);
+
+    // Create user in the database (using Sequelize ORM)
+    const user = await db.User.create({
+      name,
+      email,
+      roleId: 2,
+      password: hashedPassword,
+    });
+    res.status(200).json({ message: user }); // /!\use jsend
```
Comment (Muhet) [inline src/controller/authController.js]: Use jsend format

**Review by Muhet — DISMISSED** [review]
I've done good job,

File: src/controller/authController.js (hunk 1 above)
Comment (bryan936M) [reply src/controller/authController.js]: Done

**Review by Muhet — APPROVED** [review]
greate

**Review by chanceDira — APPROVED** [review]
NIce work @bryan936M

**Pull request merged**